src/parser/jobs/mch/modules/Reassemble.tsx

```diff
@@ -1,86 +1,108 @@
-import {Trans, Plural} from '@lingui/react'
-import {ActionLink} from 'components/ui/DbLink'
-import {ActionRoot} from 'data/ACTIONS/root'
-import STATUSES from 'data/STATUSES'
-import {CastEvent, BuffEvent} from 'fflogs'
-import Module, {dependency} from 'parser/core/Module'
-import Combatants from 'parser/core/modules/Combatants'
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {ActionKey} from 'data/ACTIONS'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
 import {Data} from 'parser/core/modules/Data'
-import Suggestions, {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
 import React from 'react'
 
 // These are the only GCDs that should be reassembled under normal circumstances
-const REASSEMBLE_GCDS: Array<keyof ActionRoot> = [
+const REASSEMBLE_GCDS: ActionKey[] = [
 	'AIR_ANCHOR',
 	'AUTO_CROSSBOW',
 	'DRILL',
 	'SPREAD_SHOT',
+	// 6.0:
+	//'CHAIN_SAW',
+	//'SCATTER_GUN',
```

Comment:
That's a bap.

Reply:
Ja, please keep 6.0 stuff out of the 5.x branch. I'll be splitting branches shortly for additions 👍